src/main/java/mate.academy/lib/Injector.java

```diff
@@ -1,13 +1,60 @@
 package mate.academy.lib;
 
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+import mate.academy.service.FileReaderService;
+import mate.academy.service.ProductParser;
+import mate.academy.service.ProductService;
+import mate.academy.service.impl.FileReaderServiceImpl;
+import mate.academy.service.impl.ProductParserImpl;
+import mate.academy.service.impl.ProductServiceImpl;
+
 public class Injector {
     private static final Injector injector = new Injector();
+    private static final Map<Class<?>, Class<?>> interfaceClass = new HashMap<>();
```

Comment:
The variable name 'interfaceClass' is not very informative. Consider renaming it to something like 'interfaceImplementations' to better reflect its purpose.

Reply:
changes have been made